MdeModulePkg/Core/Dxe/DxeMain/DxeMain.c

```diff
@@ -244,6 +244,7 @@ DxeMain (
   EFI_VECTOR_HANDOFF_INFO       *VectorInfoList;
   EFI_VECTOR_HANDOFF_INFO       *VectorInfo;
   VOID                          *EntryPoint;
+  VOID                          *Ptr; // MU_CHANGE

```

Comment:
Instead of using VOID ptr it might be better to use something like
 EFI_HOB_GUID_TYPE       *GuidHob;

Reply:
Updated